src/main/java/ninjabrainbot/gui/frames/OptionsFrame.java

```diff
@@ -172,6 +173,47 @@ public int getTextSize(SizePreference p) {
 		return mainPanel;
 	}
 
+	private JPanel getHighPrecisionPanel() {
+		JPanel mainPanel = new JPanel();
+		mainPanel.setOpaque(false);
+		mainPanel.setLayout(new GridLayout(1, 2));
+		mainPanel.setBorder(new EmptyBorder(PADDING, PADDING, PADDING, PADDING));
+		JPanel column1 = new JPanel();
+		JPanel column2 = new JPanel();
+		column1.setOpaque(false);
+		column2.setOpaque(false);
+		mainPanel.add(column1);
+		mainPanel.add(column2);
+		column1.setLayout(new BoxLayout(column1, BoxLayout.Y_AXIS));
+		column2.setLayout(new BoxLayout(column2, BoxLayout.Y_AXIS));
+
+		// Tall Res Column
+		column1.add(new CheckboxPanel(styleManager, I18n.get("settings.tall_resolution"), preferences.useTallRes));
+		resolutionHeight = new FloatPreferencePanel(styleManager, I18n.get("settings.resolution_height"), preferences.resolutionHeight);
+		resolutionHeight.setDecimals(0);
+		resolutionHeight.setEnabled(preferences.useTallRes.get());
+		column1.add(resolutionHeight);
+		column1.add(new Box.Filler(new Dimension(0,0), new Dimension(0,Short.MAX_VALUE),
+				new Dimension(0, Short.MAX_VALUE)));
+
+		// Precise Sens Column
+		column2.add(new CheckboxPanel(styleManager, I18n.get("settings.use_precise_angle"), preferences.usePreciseAngle));
```

Comment:
Do you think anyone will use boat throws without tall res? On its own theres not much benefit to it if i have understood correctly, maybe it should only be enabled when tall res is turned on.

Reply:
Probably not, tall res has greater effect, is easier, and is less intrusive to the run. Plus it would simplify some other logic if boat throws required tall res. I'll make this change.